FlowCytometryTools/core/bases.py

```diff
@@ -368,6 +368,14 @@ def apply(self, func, applyto="measurement", noneval=nan, setdata=False):
 
 
 Well = Measurement
+import collections
+try:
+    from collections import abc
+    collections.MutableMapping = abc.MutableMapping
+    collections.Iterable = abc.Iterable
```

Comment:
@borcuttjahns, are these transfers from `abc.X` to `collections.X` needed somewhere? I couldn't find a place where they are used.

Reply:
Hello, I am having issues running this code and it seams to be localized to this same issue you are talking about here. I was wondering if you found a solution.

Reply:
@DavidGorlin this branch should fix the issue. It's what we currently use.